### PR TITLE
Navigation slider

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have added the below components in this PR:
1. Navigation bar
2. Drawer
3. Price slider
4. Carousel (image slider)

Please review and suggest

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -49,4 +49,110 @@ iconToastDemo.addEventListener('click', () => {
...
+            rangeInputField[0].value = minVal;
+            rangeInputField[1].value = maxVal;
+            range.style.left = ((minVal / rangeInput[0].max) * 100) + "%";
+            range.style.right = 100 - (maxVal / rangeInput[1].max) * 100 + "%";
+        }
+    });
+});
+
+// -----------------------carousel------------------
+var slidePosition = 1;
```
Comment (Maianki) [inline script.js]: Try using const or let instead of var.

File: script.js [hunk 2]
```diff
@@ -49,4 +49,110 @@ iconToastDemo.addEventListener('click', () => {
...
+}
+
+function currentSlide(n) {
+    CarouselSlideShow(slidePosition = n);
+}
+
+function CarouselSlideShow(n) {
+    var i;
+    var slides = document.querySelectorAll(".container");
+    var circles = document.querySelectorAll(".dots");
```
Comment (Maianki) [inline script.js]: Here also you can use const or let.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested a few changes. Everything else looks good. Also, can you please check your price slider component in Mozilla? I think it is not working as expected because you are using chrome-specific CSS (--WebKit prefix ) to customize it.

**Review by Anurag021 — APPROVED** [review]
code is clean, looks good to me.

File: components/navigation.css [hunk 3]
```diff
@@ -76,7 +75,7 @@ header {
     flex-direction: column;
```
Comment (MayankKumar10) [inline components/navigation.css]: use helper class for flex so you don't have to define repeat it

**Review by MayankKumar10 — APPROVED** [review]
Else everything looks good to me

File: components/navigation.css (hunk 3 above)
Comment (ApurvaSawant11) [reply components/navigation.css]: Sure. Will add more helper classes. Thanks!

File: script.js (hunk 2 above)
Comment (ApurvaSawant11) [reply script.js]: I tried using const and let, but it gave scope errors. That's why I used var. I will fix this once all of my projects are completed. I hope that's okay.

File: script.js (hunk 1 above)
Comment (ApurvaSawant11) [reply script.js]: Sure. I will fix it

**Comment by ApurvaSawant11** [comment]
> Suggested a few changes. Everything else looks good. Also, can you please check your price slider component in Mozilla? I think it is not working as expected because you are using chrome-specific CSS (--WebKit prefix ) to customize it.

Okay, I will add CSS for Mozilla as well. Thanks for pointing this out.

File: components/navigation.css (hunk 3 above)
Comment (MayankKumar10) [reply components/navigation.css]: your welcome Apurva.

File: script.js (hunk 2 above)
Comment (Maianki) [reply script.js]: sure, no issues.

**Review by Maianki — APPROVED** [review]

File: script.js (hunk 2 above)
Comment (ApurvaSawant11) [reply script.js]: I have changed the declarations. Thanks

**Pull request merged**